website/content/v1.6/kubernetes-guides/configuration/cluster-endpoint.md

```diff
@@ -1,47 +0,0 @@
----
-title: "Cluster Endpoint"
-description: "How to explicitly set up an endpoint for the cluster API"
-alises:
-  - ../../guides/configuring-the-cluster-endpoint
----
-
-In this section, we will step through the configuration of a Talos based Kubernetes cluster.
-There are three major components we will configure:
-
-- `apid` and `talosctl`
-- the controlplane nodes
-- the worker nodes
-
-Talos enforces a high level of security by using mutual TLS for authentication and authorization.
```

Comment:
this was pretty strange document, and not sure what it adds to the stuff we already have